app/forms/gpo_verify_form.rb

```diff
@@ -77,6 +80,10 @@ def pending_in_person_enrollment?
     pending_profile&.proofing_components&.[]('document_check') == Idp::Constants::Vendors::USPS
   end
 
+  def threatmetrix_check_failed?
+    pending_profile&.proofing_components&.[]('threatmetrix_review_status') == 'reject'
```

Comment:
Should this be set to `...threatmetrix_review_status != pass` instead? To account for 'reject' and 'review'?

Reply:
Yes, that's a good catch!